crates/circuit/src/parameter_table.rs

```diff
@@ -10,7 +10,7 @@
 // copyright notice, and modified files need to carry a notice indicating
 // that they have been altered from the originals.
 
-use std::cell::OnceCell;
+use std::sync::OnceLock;
```

Comment:
Is this because of free-threading support in PyO3, or a separate behavioral change?

Reply:
Yeah, because we need the members of a pyclass need to support be used in a threading context this was needed to make `CircuitData` as a pyclass work with pyo3 0.23